src/reducers/columnStateHelper.js

```diff
@@ -36,10 +36,11 @@ function initialize(state, props, oldProps) {
     scrollX = scrollLeft;
   }
 
+  const scrollXAfterScrollLeft = scrollX;
   scrollX = scrollTo(state, props, oldProps.scrollToColumn, scrollX);
 
-  // if scrollX changed due to scrollLeft or scrollToColumn, then set scrollJumpedX
-  const scrollJumpedX = scrollX != state.scrollX;
+  // if scrollX changed due to scrollToColumn, then set scrollJumpedX
+  const scrollJumpedX = scrollX != scrollXAfterScrollLeft;
```

Comment:
Earlier we set scrollJumpedX to true even when scrollLeft made a change in scrollX.
Just noticed this wasn't needed due to the call in `componentWillReceiveProps`.

Reply:
Which call in componentWillReceiveProps specifically?  How does it mitigate the need for this here?

Reply:
We use [`componentWillReceiveProps` to check for changes in `scrollLeft` and `scrollRight`](https://github.com/schrodinger/fixed-data-table-2/blob/v1.0-beta/src/FixedDataTable.js#L574), and then fire a scroll start event.
But `scrollJumpedX` can also be true due to this change in `scrollLeft`, which leads to another scroll start event.

So to fix my mistake, I only check for changes in `scrollLeft` via `scrollToColumn`

Should I update anything here with comments or better variable names?

Reply:
Ok cool, that makes sense.  Do we ever expect to have both handlers triggered by _didControlledScroll and handlers triggered by _didScrollJump for the same update?

We may want to add a comment in componentWillRecieveProps clarifying that.  and if we don't expect we'll have both ever, we may want to use an if else statement

Reply:
@wcjordan Yes, your concern was right. Moved the `_didScrollJump` logic into `_didControlledScroll` and fixed it. (anyway scroll jumps are controlled).
Please check 3960e3d.